Template/base.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
```

Comment:
継承先のhtmlファイルでtitleを指定する場合は、
```suggestion
    <title>{% block title %}{% endblock %}</title>
```
このように指定して、タイトルを指定するのがいいと思います
特にタイトルを指定する方針がなければこの行は削除していいと思います